lib/leanplum_api/api.rb

```diff
@@ -1,58 +1,53 @@
 module LeanplumApi
   class API
+    EXPORT_PENDING = 'PENDING'.freeze
+    EXPORT_RUNNING = 'RUNNING'.freeze
+    EXPORT_FINISHED = 'FINISHED'.freeze
```

Comment:
let's not move this up here.  leave it after `extend` and the `class` declaration.